pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TenantsBase.java

```diff
@@ -220,10 +222,21 @@ public void deleteTenant(
         }
     }
 
+    private Set<String> getNonBlankClusters(Set<String> allowedClusters) {
```

Comment:
What's the purpose of this check? 

If a caller passes a list like `["", ""]` the request will fail because the empty string is not a valid cluster in `allowedClusters`.

Reply:
This check is only used to make the request with blank clusters valid，i.e. use `new TenantInfo()` for create/update.